grommet/grommet
# Allow theming for TextArea, TextInput stories - Issue 5735

What does this PR do?
Working on #5735, theming support for following components is added:


- [x] TextArea
- [x] TextInput

The theme can now be switched between the currently available themes: base, grommet, hpe.

Where should the reviewer start?
Start in src/js/components and look at the changes made to the following folders:

- [x] TextArea
- [x] TextInput

What testing has been done on this PR?
Passed the existing tests ran through jest and manually verified the changes on storybook.

How should this be manually tested?
Run storybook and navigate to the following stories: TextArea, TextInput. Themes can now be switched for these stories using the Theme button.

What are the relevant issues?
https://github.com/grommet/grommet/issues/5735

Do the grommet docs need to be updated?
No.

Should this PR be mentioned in the release notes?
No.

Is this change backwards compatible or is it a breaking change?
Not a breaking change.
